pkg/v1/remote/descriptor.go

```diff
@@ -107,7 +107,11 @@ func Head(ref name.Reference, options ...Option) (*v1.Descriptor, error) {
 		return nil, err
 	}
 
-	return f.headManifest(o.context, ref, allManifestMediaTypes)
+	return f.head(o.context, ref)
+}
+
+func (f *fetcher) head(ctx context.Context, ref name.Reference) (*v1.Descriptor, error) {
+	return f.headManifest(ctx, ref, allManifestMediaTypes)
 }
```

Comment:
This method doesn't add much, can we just have callers pass `allManifestMediaTypes` themselves?

Reply:
Done.